app/build.gradle

```diff
@@ -2,8 +2,9 @@ plugins {
     id 'com.android.application'
     id 'kotlin-android'
     id 'kotlin-kapt'
+    id 'androidx.navigation.safeargs'
```

Comment:
Also. Pay attention to which plugin you add. [Java vs Kotlin](https://developer.android.com/guide/navigation/navigation-pass-data#Safe-args)